docs/kernel.rst

```diff
@@ -0,0 +1,25 @@
+Linux kernel maintenance
+========================
+
+We build and publish our own Linux kernels with additional
+`grsecurity hardening patches`_.
+The `kernel-builder`_ repository contains scripts that fetches upstream
+kernel tarballs plus grsecurity patches and produces Debian packages.
+
+Testing a new kernel
+--------------------
+
+The following steps should be performed for all of the `recommended hardware`_:
+
+#. Install the new kernel packages on your *Monitor Server*, reboot. Verify with ``uname -r`` that you are using the new kernel.
+#. If it doesn't boot, see the `Troubleshooting Kernel Updates`_ documentation.
+#. Verify ``paxtest`` returns expected results.
+#. Verify the `spectre-meltdown-checker`_ returns expected results.
+#. Upgrade your *Application Server* to the new kernel and reboot.
+#. Run basic smoke tests of SecureDrop, verify you can send a submission and a journalist can reply.
```

Comment:
We should clarify how to run basic smoke tests here, or at least link out to more details.

Also, I might suggest a slight adjustment to the wording just to make it flow a bit better:

"Run basic smoke tests of SecureDrop, then verify you can send a submission and that a journalist can reply."

Reply:
Those are supposed to be the smoke tests. Switched to "Run basic smoke tests of SecureDrop by verifying you can send a submission..."

Reply:
Ah, perfect! Much clearer now, thanks. :)